projects/storefrontlib/src/cms-components/cart/cart-coupon/applied-coupons/applied-coupons.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { I18nTestingModule, Voucher } from '@spartacus/core';
+import { AppliedCouponsComponent } from './applied-coupons.component';
+describe('AppliedCouponsComponent', () => {
+  let component: AppliedCouponsComponent;
+  let fixture: ComponentFixture<AppliedCouponsComponent>;
+
+  const mockVoucher: Voucher = {
+    code: 'MockVoucer',
+  };
+
+  const mockVouchers: Voucher[] = [mockVoucher];
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [I18nTestingModule],
+      declarations: [AppliedCouponsComponent],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AppliedCouponsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+

```

Comment:
need add two cases:
1. vouchers is undefined - coupons label is not exist
2. vouchers is 0 length - coupons label is not exist
3. vouchers has more than 1 voucher - verify for loop